molviewspec/app/api/examples.py

```diff
@@ -377,6 +546,20 @@ async def validation_data(id: str) -> Response:
     return JSONResponse(transformed_data)
 
 
+@router.get("/data/basic-primitives")
+async def basic_primitives_data() -> Response:
+    """
+    Create example primitive data.
+    """
+    builder = create_builder().primitives(default_tooltip="Triangle")
+    (
+        builder.line(start=(0, 0, 0), end=(1, 0, 0), color="red", tooltip="A"),
+        builder.line(start=(0, 0, 0), end=(0.5, (1 - 0.5**2) ** 0.5, 0), color="green", tooltip="B"),
+        builder.line(start=(1, 0, 0), end=(0.5, (1 - 0.5**2) ** 0.5, 0), color="blue"),
+    )
+    return JSONResponse(builder.as_data().dict())
```

Comment:
Made it so the builder can be used the construct the JSON data as well (can be done via the data model directly too), but the builder defines some default values so thought this would be nicer.